tests/rgen/Microsoft.Macios.Transformer.Tests/Microsoft.Macios.Transformer.Tests.csproj

```diff
@@ -11,6 +11,14 @@
         <DotnetBuildDirectory>build/dotnet</DotnetBuildDirectory>
     </PropertyGroup>
 
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+      <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+      <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    </PropertyGroup>
```

Comment:
Just an idea, but we did something like this in Android:

* https://github.com/dotnet/android/commit/d3cde470019a15af433a58a7c8a2e7ebd71e632a

So, we did this across the whole repo for Release builds or CI. There are a few projects with warnings we hope to slowly fix.

Reply:
Nice one, this project has no warnings atm but we should do something similar if we need to in the other project. I think Rolf and I have all warnings fixed or disabled with pragmas.